contrib/verify-binaries/verify.py

```diff
@@ -481,9 +485,8 @@ def print_output(
     else:
         for filename in files_to_hashes:
             print(f"VERIFIED: {filename}")
-        if missing_files:
-            for filename in missing_files:
-                print(f"MISSING: {filename}")
+        for filename in missing_files:
```

Comment:
a6dcf2cb0817143beacf0b9004191a0dd6d2e43c: did you mean to do this in the refactor commit?

Reply:
Thanks fixed in next push